include/LMS1xx/colaa_structs.h

```diff
@@ -0,0 +1,426 @@
+#ifndef COLAA_STRUCTS_H
+#define COLAA_STRUCTS_H
+
+#include "LMS1xx/parse_helpers.h"
+#include <vector>
+
+namespace CoLaAStatus
+{
+/**
+ * @brief Sensor status as returned by CoLaA::query_status()
+ */
+enum Status : uint8_t
+{
+  Undefined = 0,
+  Initialisation = 1,
+  Configuration = 2,
+  Idle = 3,
+  Rotated = 4,
+  InPreparation = 5,
+  Ready = 6,
+  ReadyForMeasurement = 7,
+  Error = 8 // Not from spec, failed to read status at all
+};
+}
+
+namespace CoLaASopasError
+{
+/**
+ * @brief Error codes returned by the sensor
+ *
+ * Used by CoLaA::read_back() and logged to console.
+ */
+enum SopasError
+{
+  Sopas_Ok = 0,
+  Sopas_Error_METHODIN_ACCESDENIED = 1,
+  Sopas_Error_METHODIN_UNKNOWNINDEX = 2,
+  Sopas_Error_VARIABLE_UNKNONWINDEX = 3,
+  Sopas_Error_LOCALCONDITIONFAILED = 4,
+  Sopas_Error_INVALID_DATA = 5,
+  Sopas_Error_UNKNOWN_Error = 6,
```

Comment:
The sudden CamelCase at the end is also in the spec sheet...

![screenshot - 12 05 2017 - 15 01 55](https://cloud.githubusercontent.com/assets/28302109/25999138/441bf32c-3724-11e7-8010-2608e79e1175.png)